client/src/modules/stock/movements/modals/search.modal.html

```diff
@@ -60,9 +60,9 @@
             <label class="control-label" translate>STOCK.FLUX</label>
             <bh-clear on-clear="$ctrl.clear('flux_id')"></bh-clear>
 
-            <ui-select name="inventory" ng-model="$ctrl.searchQueries.flux_id">
+            <ui-select multiple name="inventory" ng-model="$ctrl.selectedFluxes">
               <ui-select-match>
-                <span translate>{{$select.selected.label}}</span>
+                <span translate>{{$item.label}}</span>
```

Comment:
Could you try to set `append-to-body="true"` to the `ui-select` because the ui is not well.

![append to body](https://user-images.githubusercontent.com/5445251/30815500-521e7722-a20b-11e7-993b-0eff97d1e8dc.png)

Reply:
the dropdow list disappear if I try to add append-to-body="true" to the ui-select